Cargo.toml

```diff
@@ -66,19 +66,19 @@ split-debuginfo = "unpacked"
 #=============== Dependencies ==============#
 [workspace.dependencies]
 # Workspace members
-ferrumc-core = { path = "src/core" }
-ferrumc-ecs = { path = "src/ecs" }
-ferrumc-events = { path = "src/events" }
-ferrumc-net = { path = "src/net" }
-ferrumc-net-encryption = { path = "src/net/encryption" }
-ferrumc-net-packets = { path = "src/net/packets" }
-ferrumc-plugins = { path = "src/plugins" }
-ferrumc-storage = { path = "src/storage" }
-ferrumc-utils = { path = "src/utils" }
-ferrumc-profiling = { path = "src/utils/profiling" }
-ferrumc-logging = { path = "src/utils/logging" }
-ferrumc-macros = { path = "src/derive_macros" }
-ferrumc-world = { path = "src/world" }
+ferrumc-core = { path = "src/lib/core" }
+ferrumc-ecs = { path = "src/lib/ecs" }
+ferrumc-events = { path = "src/lib/events" }
+ferrumc-net = { path = "src/lib/net" }
+ferrumc-net-encryption = { path = "src/lib/net/encryption" }
+ferrumc-net-packets = { path = "src/lib/net/packets" }
+ferrumc-plugins = { path = "src/lib/plugins" }
+ferrumc-storage = { path = "src/lib/storage" }
+ferrumc-utils = { path = "src/lib/utils" }
+ferrumc-profiling = { path = "src/lib/utils/profiling" }
+ferrumc-logging = { path = "src/lib/utils/logging" }
+ferrumc-macros = { path = "src/lib/derive_macros" }
+ferrumc-world = { path = "src/lib/world" }
```

Comment:
Can we tab these ?

Reply:
This is a test comment, you can close it by hitting the *Resolve* button.

Reply:
tab what

Reply:
Like what ReCore deleted at these lines. Also. You should mark *Resolved* when you actually resolved it. Otherwise the discussion is hidden.

https://github.com/ferrumc-rs/ferrumc/commit/50bb2fd5f6c2f9ddcfd9aa38de6399c1573cad9d#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L66-L75

Reply:
But like you can ignore it was just a test. We can continue as it is now